examples/TID_model_custom_property/Modelling_TID_TNID_example_notebook.ipynb

```diff
@@ -0,0 +1,471 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    cfg.sim.start_time=today.mjd2000 * pk.DAY2SEC # convert epoch to seconds
</pre><p>maybe confusing because today is actually not today, maybe just start_date?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aidotse/PASEOS/pull/168/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>